### ISSUE TITLE
Search filter

### ISSUE THREAD

**Issue by pengweichu** [sipcapture/homer#488]
The button "OR" "AND" seems not affected,  it's always AND here even set it to OR/AND, what's do I missed?

![image](https://user-images.githubusercontent.com/12697283/132948849-9cb09717-a105-4660-bc2a-7f403b837cc2.png)

**Comment by pengweichu** [comment]
Sorry for my stuip questoin.

**Comment by adubovikov** [comment]
confirmed

**Comment by adubovikov** [comment]
fixed here https://github.com/sipcapture/homer-app/commit/e80016705dac51cab142b0ffa3b4a25a3863af13

the package 1.4.11 is compiling

**Comment by pengweichu** [comment]
> fixed here [sipcapture/homer-app@e800167](https://github.com/sipcapture/homer-app/commit/e80016705dac51cab142b0ffa3b4a25a3863af13)
> 
> the package 1.4.11 is compiling

Seems not works for me.
I installed HOMER by `homer_installer.sh` yesterday, and today I pulled the homer-app repo then compile it, use new homer-app to replace the /usr/local/bin/homer-app, then start the service again succeeded, but I observed the below behaviors:

1. The AND and OR all are not affected to the old SIP messages, even I entered the CALLID to filter, all SIP messages are listed.
2. New SIP messages can't be queried, always display blank.

Can I simply repeat the Bash Installer steps to install it again?

**Comment by pengweichu** [comment]
> fixed here [sipcapture/homer-app@e800167](https://github.com/sipcapture/homer-app/commit/e80016705dac51cab142b0ffa3b4a25a3863af13)
> 
> the package 1.4.11 is compiling

Thanks for your fix, below is the feedback.
Yesterday I installed the HOMER by Bash installer, today I pull the homer-app, compile and replaced /usr/local/bin/homer-app,
I observed the below behaviors:

1. The new SIP messages send from my server to HOMER can't be quired, even don't set any filter just choose the time range to last 3 hours. But no problem to query the previous SIP messages which were sent yesterday.
2. I set the AND logic for the filters, works fine, get the 5 SIP messages to display there. Then I switch to OR logic for filters, search again, but the filters not affected, all SIP messages are displayed even the messages don't meet any filter. After then, I click the button to switch the logic to AND, search again, all SIP messages are displayed(I expect only display 5 SIP messages as the first time search), even the messages don't meet any filter. I must refresh the whole page by F5 and search again to get the AND logic to work fine.
3. The OR logic does not work anyway.

Should I install the HOMER by BASH INSTALLER again to instead compile the Homer-app and replace it manually?
